### PR TITLE
AdvancedMicroDevices,Inc.AMDRyzenMaster version 2.10.0.2227

### PR DESCRIPTION
**Pull request by Debtless**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89598)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by Trenly** [comment]
@msftbot dependencies

File: manifests/a/AdvancedMicroDevices,Inc/AMDRyzenMaster/2.10.0.2227/AdvancedMicroDevices,Inc.AMDRyzenMaster.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AdvancedMicroDevices,Inc.AMDRyzenMaster
+PackageVersion: 2.10.0.2227
+Installers:
```
Comment (stephengillie) [inline manifests/a/AdvancedMicroDevices,Inc/AMDRyzenMaster/2.10.0.2227/AdvancedMicroDevices,Inc.AMDRyzenMaster.installer.yaml:6]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
InstallModes:
- interactive
- silent
Installers:
```

File: manifests/a/AdvancedMicroDevices,Inc/AMDRyzenMaster/2.10.0.2227/AdvancedMicroDevices,Inc.AMDRyzenMaster.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AdvancedMicroDevices,Inc.AMDRyzenMaster
+PackageVersion: 2.10.0.2227
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download.amd.com/Desktop/AMD-Ryzen-Master.exe
+  InstallerSha256: E998DDBBA7AF72D4D6AE9E673E668C0E272D5646078A8119E429EB12D7657560
```
Comment (stephengillie) [inline manifests/a/AdvancedMicroDevices,Inc/AMDRyzenMaster/2.10.0.2227/AdvancedMicroDevices,Inc.AMDRyzenMaster.installer.yaml:10]:
```suggestion
  InstallerSha256: E998DDBBA7AF72D4D6AE9E673E668C0E272D5646078A8119E429EB12D7657560
  InstallerSwitches:
    Silent: /S /v/q
```

**Pull request closed without merging**